### PR TITLE
Update forms.rst

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Just some minor improvements. More to come...

### CODE REVIEW

**Comment by ThomasLandauer** [comment]
Question: How can I change the order of the "Learn more"-links https://symfony.com/doc/3.4/forms.html#learn-more ? What does this placeholder mean: `/form/*`?

**Comment by xabbuh** [comment]
Is it really necessary to move so much stuff around? This makes it quite hard to review the changes.

**Comment by ThomasLandauer** [comment]
> Is it really necessary to move so much stuff around?

Yes!!! :-)

File: forms.rst [hunk 1]
```diff
@@ -449,47 +424,6 @@ the date as a string in the box)::
 .. image:: /_images/form/simple-form-2.png
     :align: center
 
-Each field type has a number of different options that can be passed to it.
-Many of these are specific to the field type and details can be found in
-the documentation for each type.
-
-.. sidebar:: The ``required`` Option
```
Comment (xabbuh) [inline forms.rst]: What is the reason to remove this section?

File: forms.rst [hunk 2]
```diff
@@ -449,47 +424,6 @@ the date as a string in the box)::
...
-            'widget' => 'single_text',
-            'label'  => 'Due Date',
-        ])
-
-    The label for a field can also be set in the template rendering the
-    form, see below. If you don't need a label associated to your input,
-    you can disable it by setting its value to ``false``.
-
 .. index::
    single: Forms; Field type guessing
```
Comment (xabbuh) [inline forms.rst]: I would move the whole section about form type (and type options) guessing into a dedicated article. Handling this here in the main article about forms can be confusing.

File: forms.rst (hunk 1 above)
Comment (ThomasLandauer) [reply forms.rst]: Well, doesn't contain any relevant information. The documentation is linked anyway.

File: forms.rst (hunk 2 above)
Comment (ThomasLandauer) [reply forms.rst]: Well, I don't say that this is the perfect solution for everything. I'm just saying that it's better than it was ;-)

**Comment by HeahDude** [comment]
I'm really sorry but the changes are almost impossible to review or merge up into master due to changes in all branches since you've opened that PR.
So I'm going to close here, feel free to propose your changes by opening a new PR targeting master, thank you for understanding.

**Pull request closed without merging**